demos/demo9/index.tsx

```diff
@@ -46,8 +46,8 @@ export default () => {
 	[node1, node2, link1, link2].forEach((item) => {
 		item.addListener({
 			selectionChanged: (node, isSelected) => {
```

Comment:
should it not actually be: `selectionChanged : action('selectionChanged')`

Reply:
Makes sense totally